packages/api-client/src/providers.ts

```diff
@@ -56,7 +56,7 @@ export const deleteProvider = async (params: {
     payload: {
       httpMethod: 'DELETE',
       resource: '/{proxy+}',
-      path: `/providers/${providerId}`,
+      path: `/providers/${encodeURIComponent(providerId)}`,
```

Comment:
Same comment as above, we could probably use a path helper here for consistency. Annoying comment, I'm sorry.

Reply:
See my above longer answer